ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -24,6 +24,7 @@
   ```yaml
   # Use at the host level
   is_deployed: < true | false | default -> true >
+  ```
```

Comment:
Will do. Going to use the PR to add more doc fixes I've been meaning to get to. Thank you Tony

Reply:
This one is planned to be fixed by @ClausHolbechArista  in https://github.com/aristanetworks/ansible-avd/pull/1826/files#diff-150fb45070956c50bd6bbd72322e4d6f716d8d7e9c66a1162bad5e29247ea9dcR27 though it might take some time to merge so we probably can fix it in this one.